src/main/java/org/folio/utils/UserPermissionsUtil.java

```diff
@@ -24,19 +24,22 @@
 
 import io.vertx.core.json.JsonArray;
 import org.folio.rest.jaxrs.model.Invoice;
+import org.folio.rest.jaxrs.model.Parameter;
 
 public final class UserPermissionsUtil {
-  public static final String OKAPI_HEADER_PERMISSIONS = "X-Okapi-Permissions";
 
+  public static final String OKAPI_HEADER_PERMISSIONS = "X-Okapi-Permissions";
   private static final String EMPTY_ARRAY = "[]";
 
   private UserPermissionsUtil() {
-
   }
 
   public static void verifyUserHasAssignPermission(List<String> acqUnitIds, Map<String, String> okapiHeaders) {
     if (CollectionUtils.isNotEmpty(acqUnitIds) && isUserDoesNotHaveDesiredPermission(ASSIGN, okapiHeaders)) {
-      throw new HttpException(HttpStatus.HTTP_FORBIDDEN.toInt(), USER_HAS_NO_ACQ_PERMISSIONS);
+      var param1 = new Parameter().withKey("Assign").withValue(ASSIGN.getPermission());
+      var param2 = new Parameter().withKey("User permission").withValue(okapiHeaders.get(OKAPI_HEADER_PERMISSIONS));
+      var param3 = new Parameter().withKey("acqUnitIds").withValue(acqUnitIds.toString());
```

Comment:
I think these params will not help us, we can revert this file

Reply:
Thanks, revereted this file